sspl_test/constants.sh

```diff
@@ -16,4 +16,7 @@
 PRODUCT_NAME='LDR_R2'
```

Comment:
do we need this file even now, given all setup py scripts will be using sspl_constants.py

Reply:
yes, in run_qa_tests.sh  we are fetching data from this file.

Reply:
Ok